frontend/workflows/experimentation/src/index.tsx

```diff
@@ -18,6 +19,13 @@ const register = (): WorkflowConfiguration => {
         description: "List Experiments.",
         component: ListExperiments,
       },
+      viewExperimentRun: {
```

Comment:
@dschaller @mcutalo88 Is it possible for one route (`listExperiments` from line 16.) to require the existance of another route (`viewExperimentRun`)?